README.md

```diff
@@ -161,13 +156,13 @@ edged:
     version: 2.0.0
 ```
 
-Run
+### Run
 
 ```shell
 # run mosquitto
 mosquitto -d -p 1883
+
 # run edge_core
-cd cmd/
 ./edge_core
```

Comment:
Curious about why binary generated in mainly directory, rather than in `cmd/`. Can we change this?